doc/modules/learning_curve.rst

```diff
@@ -103,6 +103,24 @@ the parameter :math:`\gamma` of an SVM on the digits dataset.
    :align: center
    :scale: 50%
 
+If you intend to plot the validation curves only, the class
```

Comment:
I would move this figure right after the code snippet above and move the `If the training score and the validation score are both low` after the new plot.

Otherwise, it's weird: we compute the validation curves of the `C` parameter of a `SVC(kernel="linear")` model on iris using the `validation_curve` function, then discuss how to analyze the values using the figure of a plot that studies the `gamma` parameter of `SVC(kernel="rbf")` on another dataset from an example and then come back with another plot on the original model and dataset which is quite confusing.

It's based to avoid the back and forth switching between the 2 model/dataset pairs by reordering those paragraphs as suggested.